integration/kubernetes/vfio.sh

```diff
@@ -124,21 +124,27 @@ run_test() {
 
 	setup_configuration_file "$image_type" "$machine_type" "$hypervisor" "$sandbox_cgroup_only"
 
+	pod_name=pod$(echo ${RANDOM}${RANDOM})
+	sed "s/@POD_NAME@/${pod_name}/" ${SCRIPT_DIR}/runtimeclass_workloads/vfio.yaml.in > ${SCRIPT_DIR}/runtimeclass_workloads/vfio.yaml
 	sudo -E kubectl create -f "${SCRIPT_DIR}/runtimeclass_workloads/vfio.yaml"
-
-	pod_name=vfio
 	sudo -E kubectl wait --for=condition=Ready pod "${pod_name}" || \
 		{
 			sudo -E kubectl describe pod "${pod_name}";
 			die "Pod ${pod_name} failed to start";
 		}
 
 	# wait for the container to be ready
-	waitForProcess 15 3 "sudo -E kubectl exec ${pod_name} -- ip a"
+	for iii in $(seq 1 30); do
+		mac_addrs=$(sudo -E kubectl exec "${pod_name}" -- ip a)
+		[ "${mac_addrs}" != "" ] && break
+	done
+	info "Got mac_addrs for ${pod_name}: ${mac_addrs}"
+
 
 	# Expecting 2 network interaces -> 2 mac addresses
-	mac_addrs=$(sudo -E kubectl exec "${pod_name}" -- ip a | grep "link/ether" | wc -l)
-	if [ ${mac_addrs} -ne 2 ]; then
+	mac_addrs_count=$(echo "${mac_addrs}" | grep "link/ether" | wc -l || true)
+	if [ ${mac_addrs_count} -ne 2 ]; then
+		sudo -E kubectl describe pod "${pod_name}" || tru
```

Comment:
`tru` -> `true`.
Are you using the same Mac like me? As the `e` is often lost :-)

Reply:
good catch, thanks